lib/killbill_client/models/combo_transaction.rb

```diff
@@ -34,14 +36,18 @@ def combo_payment(user, reason, comment, options, refresh_options = nil)
                                                 }.merge(options)
         rescue KillBillClient::API::ResponseError => error
           response = error.response
-          if response.header['location']
+          if follow_location && response.header['location']
             created_transaction = ComboTransaction.new
             created_transaction.uri = response.header['location']
           else
             raise error
           end
         end
-        created_transaction.refresh(refresh_options || options, Payment)
+
+        if follow_location
+          return created_transaction.refresh(refresh_options || options, Payment)
+        end
+        created_transaction
```

Comment:
Maybe a better option here would be returning `nil`, since it can get confusing:
* Return a `Payment` instance when `:follow_location` is true
* Otherwise, return a `Transaction`/`ComboTransaction` with the `uri` attribute set